static/js/controllers/edit-user.js

```diff
@@ -172,37 +162,25 @@ var modal = require('../modules/modal');
         };
       };
 
-      var updateComplete = function(pane, err) {
-        if (err) {
-          pane.done($translate.instant('Error updating user'), err);
-          return;
-        }
-        if ($scope.editUserModel.password) {
-          // reload the page so the user can log in with the new password
-          $window.location.reload(true);
-        }
-        $rootScope.$broadcast('UsersUpdated', $scope.editUserModel.id);
-        $scope.editUserModel = null;
-        $uibModalInstance.close('ok');
-      };
-
       $scope.updatePassword = function() {
         $scope.errors = {};
+        $scope.setProcessing();
         if (validatePassword()) {
-          var pane = modal.start($('#update-password'));
           var updates = { password: $scope.editUserModel.password };
           UpdateUser($scope.editUserModel.id, null, updates)
             .then(function() {
-              updateComplete(pane);
+              $scope.setFinished();
+              $window.location.reload(true);
```

Comment:
Yes. When updating your own password the page gets refreshed so we don't need to broadcast that the users have changed.